addons/sourcemod/scripting/scp_sf.sp

```diff
@@ -3047,11 +3047,11 @@ public Action SendProp_OnAliveMulti(int entity, const char[] propname, int &valu
 	{
 		if(IsSpec(target))
 		{
-			if(!IsValidClient(client))
+			if(!IsValidClient(client) || !IsClientInGame(client))
```

Comment:
This is not necessary as `IsValidClient()` already does a `IsClientInGame` check